predicators/nsrt_learning/strips_learning/effect_search_learner.py

```diff
@@ -4,15 +4,15 @@
 
 import abc
 import functools
-from dataclasses import dataclass
-from typing import Callable, Dict, Iterator, List, Optional, Sequence, Set, \
+from typing import Callable, Dict, FrozenSet, Iterator, List, Optional, Set, \
```

Comment:
should we call this file something else now that we're no longer just searching over effect sets?

Reply:
similarly, change all the names of classes, variables, etc. that use "effect set" but are no longer just about effect sets

Reply:
Renaming to PNAD search